### ISSUE TITLE
Always select 3d models with bones and animation.

### ISSUE THREAD

**Issue by quangtri3111996** [mrdoob/three.js#25859]
### Description

In a scene consisting of 3d model with bones and animation, when I select another 3d model in that scene, it always selects the 3d model with bones and animation. But when I try a scene with the model without bones and animation, this problem doesn't happen. Watch the video the phone is a 3d model without bones and the cheetah is a 3d model with bones and animation. 

(Please watch the video: https://drive.google.com/file/d/1DPkdwFU8fxpJ2upq7JLaToFyYSoUZpY4/view?usp=sharing)

### Reproduction steps

1. Create a scene with models and a model with bones and animation.
2. Click on any a model in the scene, it always selects to the model with bones and animation
3. Create a scene with models and a model without bones and animation.
4. This issue doesn't appear with the 3d model without bones

### Code

No code example

### Live example

https://drive.google.com/file/d/1DPkdwFU8fxpJ2upq7JLaToFyYSoUZpY4/view?usp=sharing

### Version

r151

### Browser

Chrome

**Comment by donmccurdy** [comment]
Hi @quangtri3111996! We cannot investigate this issue without information requested in the bug report form. Could you provide the model or a live demo (with code), rather than a video? It's possible this might be fixed by https://github.com/mrdoob/three.js/pull/25791, but without the information required, it won't be possible for us to find out.

**Comment by quangtri3111996** [comment]
Hi @donmccurdy, thanks for the reply. Please see the video (https://drive.google.com/file/d/1rErzPcGk3dY_bWYVW1JN2N3tjabEQxTe/view?usp=sharing)

- I have a 3d room. First, I can select new newspapers on the table, on the chair. But then I add a 3d model (a jaguar) have bones and animation, I can't select any models are close the jaguar model. When I click on the new papers or the chair, it always select to the jaguar.

**Comment by donmccurdy** [comment]
@quangtri3111996 sorry, what's missing is a way for me to recreate the problem on my own computer. I cannot debug the video, we will need access to the files you're loading. GitHub comments should allow you to attach .ZIP files if that's OK.

**Comment by quangtri3111996** [comment]
Hello @donmccurdy, any new update on this?

**Comment by donmccurdy** [comment]
It seems that there are still some issues with bounds, raycasting, and frustum culling against skinned meshes on the `dev` branch. Frustum culling isn't correct either here:

https://user-images.githubusercontent.com/1848368/233844805-1d85d34d-d6c9-48bd-9395-0dcbb6b91c7f.mov

**Comment by quangtri3111996** [comment]
Hi @donmccurdy , thanks for your answer, but I still not clear how to solve my problem. Again, I can select any pattern in the room on the viewport when I don't add the jaguar model. But when I add the jaguar model and I select the models that are close to the jaguar model, it always focuses on the jaguar model

**Comment by donmccurdy** [comment]
@quangtri3111996 Sorry, I do not have a solution to your problem right now. Raycasting against THREE.SkinnedMesh has been buggy for a while. This is something we want to fix, but it will take some time.

**Comment by quangtri3111996** [comment]
@donmccurdy thank for support

**Comment by quangtri3111996** [comment]
@donmccurdy, thanks for supporting, Hope to solve it in the near future.

**Comment by Mugen87** [comment]
I think the problem is that `expandByObject()` does not work with `SkinnedMesh` yet because it does not honor bone transformations in its `precise` code path:

https://github.com/mrdoob/three.js/blob/45304ac64b3943173f5272b134031ed36aadaf68/src/math/Box3.js#L181-L191

**Comment by Mugen87** [comment]
After more debugging this happens because the editor allows to select bones over the skeleton helper which is probably something that should be prevented.

**Comment by Mugen87** [comment]
The editor allows to select bones over the skeleton helper which is probably something that should not be allowed.

Independently, the first click was outside the group's (yellow) bounding box and it was still possible to raycast the skeleton helper. This also needs to be investigated.

**Comment by quangtri3111996** [comment]
Hello @Mugen87 and @donmccurdy any update on this?

**Comment by Mugen87** [comment]
> I'm not sure why this picker is nescessary since the lines of each helper can be raycasted against as well as the skinnedMesh itself.

You are absolutely right. I've stumbled across this issue today again and I believe the picker was initially intended for lights. Now that raycasting properly works with skinned meshes and lines can be properly raycasted as well, there is indeed no need for a picker object anymore. I'll file a PR.
